pkg/app/piped/controller/scheduler.go

```diff
@@ -220,7 +220,7 @@ func (s *scheduler) Run(ctx context.Context) error {
 	repoCfg, ok := s.pipedConfig.GetRepository(repoID)
 	if !ok {
 		s.doneDeploymentStatus = model.DeploymentStatus_DEPLOYMENT_FAILURE
-		statusReason = fmt.Sprintf("Unable to find %q from the repository list in piped config", repoID)
+		statusReason = fmt.Sprintf("Repository %q is not found in the piped config", repoID)
```

Comment:
nit, `statusDesc` or `statusDescription` can be considered as a better name 🤔

Reply:
That name was already fixed/used in the proto so let me keep it as is.

Reply:
🙆‍♀️